CI_CD/scripts/model_promotion.py

```diff
@@ -0,0 +1,89 @@
+from model import *
+from data import get_dataloader
+import neptune.new as neptune
+import os
+import subprocess
+import sys
+
+project_name = 'common/pytorch-integration'
+
+# Fetch project 
+project = neptune.get_project(
+    name = project_name,
+    api_token=os.getenv("NEPTUNE_API_TOKEN")
+)
+
+# In-prod
+# Filter in-prod run
+prod_runs_table_df = project.fetch_runs_table(tag='in-prod').to_pandas()
```

Comment:
I think there is always only one in-prod best model right?
So maybe if we fetch via tag 'champion' we would communicate the fetching of best model better (and make the logic simpler) ?

Reply:
You are correct, there is only one.

Since I already changed the docs, it makes sense.